client/src/components/DataFiles/DataFilesModals/DataFilesModalTables/DataFilesModalSelectedTable.js

```diff
@@ -16,8 +16,6 @@ const DataFilesSelectedStatusCell = ({ row, operation }) => {
   switch (status[row.original.id]) {
     case 'RUNNING':
       return <LoadingSpinner placement="inline" />;
-    case 'SUCCESS':
-      return <span className="badge badge-success">SUCCESS</span>;
```

Comment:
Since we're batching notifications now, I think it'd be helpful to keep these statuses while the modal is open